YouTube.js

```diff
@@ -166,22 +170,40 @@ var testCases = [
 		"items": [
 			{
 				"itemType": "videoRecording",
-				"title": "HD Charlie Chaplin - The Great Dictator: Speech to Humanity | HD",
+				"title": "The Great Dictator - Speech to Humanity | HD",
 				"creators": [
 					{
-						"lastName": "EMS co",
-						"creatorType": "contributor",
-						"fieldMode": 1
+						"firstName": "E. M. S.",
+						"lastName": "co",
+						"creatorType": "contributor"
```

Comment:
OK, makes sense. One of the vids I tried this on was a real name, but I agree that's rare